src/main/twirl/views/manage/viewDescription.scala.html

```diff
@@ -0,0 +1,27 @@
+@(
+  opp: Opportunity
+)
+
+@formatId(id: Long) = @{
+    f"RIFS$id%05d"
+}
+
+@main(s"${opp.title} - RIFS", backLink=Some(BackLink("Opportunity template",controllers.routes.OpportunityController.showOverviewPage(opp.id).url))) {
+    <div class="grid-row">
+        <div class="column-full">
+            <h1 class="heading-xlarge">
+                <span class="heading-secondary">@formatId(opp.id.id): @opp.title</span>
+                Opportunity description
+            </h1>
+            @opp.description.map { des =>
+            <p class="heading-secondary">@des.text</p>
```

Comment:
The class for the paragraph should be "text" so that it wraps at around 7-8 words.